Lib/configparser.py

```diff
@@ -1082,7 +1084,7 @@ def _read_inner(self, fp, fpname):
                     st.indent_level = sys.maxsize
                 continue
 
-            first_nonspace = self.NONSPACECRE.search(line)
+            first_nonspace = self.NONSPACECRE.search(line.value)
```

Comment:
I don't love that the value of the line can't be accessed directly, but must go through an attribute access. The previous string subclass gave much nicer ergonomics for a "line of text".